### DISCUSSION TITLE
Possible to surface build errors better from steps of "npm run tauri build"?

### DISCUSSION THREAD

**Discussion by dbarnett** [gulbanana/gg#12] [Q&A]
Initially when I ran `npm run tauri build`, it was failing at the end with
```
    Bundling gg_0.18.0_amd64.AppImage (/home/david/projects/gg/src-tauri/target/release/bundle/appimage/gg_0.18.0_amd64.AppImage)
    Error failed to bundle project: error running build_appimage.sh
```

It turned out to be caused by a missing librsvg dependency breaking a gtk plugin step, but it took a while to find the script and figure out how to manually run it correctly to find the actual error. Is there a way to improve the ergonomics there?

**Comment by gulbanana** [comment]
You can run just that step with `npm run tauri bundle` (or `cargo tauri bundle`), and it has a --debug option. However, I think `tauri bundle` only works if you've done the rest of the `tauri build` first 😓

**Comment by gulbanana** [comment]
Passing `--debug` through to the Tauri CLI (prefixed with some number of `--` depending on what runner you're using) is the best way to get more info here.
